packages/babel-cli/src/_babel-node.js

```diff
@@ -13,13 +12,24 @@ import pkg from "../package.json";
 
 const program = new commander.Command("babel-node");
 
+function collect(value, previousValue): Array<string> {
```

Comment:
Not a big deal, but worth creating `cli-utils` or something for this and `booleanify`?